controllers/VerifyToken.js

```diff
@@ -0,0 +1,25 @@
+var jwt = require('jsonwebtoken');
+var config = require('../config/config1');
+
```

Comment:
mode all these functionality to auth library. as it is related to authentication